csrc/transformer/general_kernels.cu

```diff
@@ -43,7 +43,7 @@ __global__ void column_sum_reduce(const T* __restrict__ inp,
 
     if (threadIdx.x == 0) {
         int pos = blockIdx.x * TILE_DIM + threadIdx.y;
-        if (pos < (rows * width)) out[pos] = sum;
+        if (pos < width) out[pos] = sum;

```

Comment:
Thanks for fixing this! I would say it still was working when the hidden dimension was dividable by 32, however, it would have a caused a memory leak for when the hidden is not dividable by 32!

Reply:
yes! thanks for your approval!!